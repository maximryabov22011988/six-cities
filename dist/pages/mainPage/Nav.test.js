'use strict';
Object.defineProperty(exports, '__esModule', { value: true });
const React = require('react');
const renderer = require('react-test-renderer');
const Nav_1 = require('./Nav');
it('Nav renders correctly with mandatory props', () => {
  const tree = renderer.create(React.createElement(Nav_1.default, null, null)).toJSON();
  expect(tree).toMatchSnapshot();
});
//# sourceMappingURL=Nav.test.js.map
