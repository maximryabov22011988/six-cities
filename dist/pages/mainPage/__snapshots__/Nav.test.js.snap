// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav renders correctly with mandatory props 1`] = `
<div
  className="cities tabs"
>
  <section
    className="locations container"
  />
</div>
`;
